### ISSUE TITLE
go get latest fails 

### ISSUE THREAD

**Issue by aranw** [go-quicktest/qt#9]
When trying to use `go get github.com/go-quicktest/qt@latest` get an error reporting that the module has been declared as `github.com/frankban/quicktest` 

```
❯ go get github.com/go-quicktest/qt@latest
go: github.com/go-quicktest/qt@v1.14.2: parsing go.mod:
        module declares its path as: github.com/frankban/quicktest
                but was required as: github.com/go-quicktest/qt
```

Work around for this is to explicitly use `go get` with a version

```
❯ go get github.com/go-quicktest/qt@v0.0.2
go: added github.com/go-quicktest/qt v0.0.2
go: upgraded github.com/google/go-cmp v0.5.5 => v0.5.7
go: upgraded github.com/kr/pretty v0.2.1 => v0.3.0
go: upgraded github.com/kr/text v0.1.0 => v0.2.0
go: added github.com/rogpeppe/go-internal v1.6.1
```

**Comment by mvdan** [comment]
This is because @rogpeppe or @frankban mistakenly pushed some old v1 tags from https://github.com/frankban/quicktest to this new repository, so proxy.golang.org is remembering them forever, unfortunately.

I think this issue should be kept open until `@latest` is fixed; I imagine the best we could do is retract those "latest" versions, now that https://github.com/golang/go/issues/53594 is fixed for the upcoming Go 1.19.

**Comment by frankban** [comment]
It seems that I was finally able to fix this with https://github.com/go-quicktest/qt/pull/13
`go get github.com/go-quicktest/qt@latest` should now successfully install `v0.1.0`.
Please reopen if you encounter further problems.
